app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,8 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
 
-    <string name="noGiftCards">Click the "+" (plus) button to add a card first.\n\nCatima carries your cards on the device, so they are always within reach.</string>
-    <string name="noMatchingGiftCards">Didn\'t find anything. Try changing your search.</string>
+    <string name="noGiftCards">Click the ⊕ plus button to add a card, or import some from the ⋮menu first.</string>
```

Comment:
Supposedly 

![bilde](https://user-images.githubusercontent.com/13802408/113522720-7e337f00-9592-11eb-9934-7c0793493672.png)
which looks more similar than just + (?)

Reply:
Yup, it is a "floating action button" so it looks exactly like that.